pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -45,8 +45,16 @@
   flex: 2;
 }
 
+.logoContainer {
+  max-height: var(--web-app-component-detail-logo-height);
+  max-width: var(--web-app-component-detail-logo-width);
+  text-align: center;
```

Comment:
Whenever you use `text-align` on something that's not a text, it's probably best to look into (e.g.)

```
display: flex;
justify-content: center;
```